app/ante_no_seq.go

```diff
@@ -59,12 +74,21 @@ func NewAnteHandler(
 		ante.TxTimeoutHeightDecorator{},
 		ante.NewValidateMemoDecorator(ak),
 		ante.NewConsumeGasForTxSizeDecorator(ak),
-		deductFeeDecorator,
+		block.NewIgnoreDecorator(
+			deductFeeDecorator,
+			blockSDKParams.freeLane,
+		),
 		ante.NewSetPubKeyDecorator(ak), // SetPubKeyDecorator must be called before all signature verification decorators
 		ante.NewValidateSigCountDecorator(ak),
 		ante.NewSigGasConsumeDecorator(ak, sigGasConsumer),
-		ante.NewSigVerificationDecorator(ak, signModeHandler),
-		// ante.NewIncrementSequenceDecorator(ak),
+		// ante.NewSigVerificationDecorator(ak, signModeHandler) <-- removed this to prevent failures resulting from invalid tx orders in e2e
+		ante.NewIncrementSequenceDecorator(ak),
```

Comment:
notice: This switch is made, as the priority nonce-mempool (underlying mempool used by the block-sdk) requires that all txs have a unique sequence number. As such, we remove the signature verification ante-handler (to ignore the sequence # check in e2e txs), but still increment the seq. to get around over-writing txs in the mempool